src/utils/src/dataset-utils.ts

```diff
@@ -586,6 +588,80 @@ export function analyzerTypeToFieldType(aType: string): string {
   }
 }
 
+/**
+ * Convert arrow data type to kepler.gl field types
+ *
+ * @param arrowType the arrow data type
+ * @returns corresponding type in `ALL_FIELD_TYPES`
+ */
+export function arrowDataTypeToFieldType(arrowType: ArrowDataType): string {
```

Comment:
Just a thought: I wonder if this could go together with the ArrowContainer? I.e. consolidate arrow knowledge as much as possible.